src/Serializer/Normalizer/ObjectNormalizer.php

```diff
@@ -139,7 +140,13 @@ private function denormalizeProperty(mixed $data, PropertyMetadata $property, ?s
             return null;
         }
 
-        $result = $this->serializer->denormalize($value, $property->getType(), $format, $context);
+        try {
+            $result = $this->serializer->denormalize($value, $property->getType(), $format, $context);
+        } catch (\Exception $t) {
+            // re-throw with additional information
+            throw new NotNormalizableValueException(sprintf($t->getMessage().' On property: %s::%s.', $context['resource_class'], $property->getName()), 0, $t);
+        }
+
```

Comment:
We can not rely on $context['resource_class'] as this would only work for API-Platform (and also not even for every possible scenario there)

Only catch and re-throw NotNormalizableValueException instead of \Exception and please use $e instead of $t.

Reply:
oh and please add a test that tests that this exception is thrown. maybe you also remmeber that I said to do it right, it would make sense to introduce a VOM-specific NotNormalizableValueException, which would make the test more valuable. If the test just tests that symfony's NotNormalizableValueException is thrown, it can not verify that the rethrow actually worked. Only a test for an expected Zolex\VOM\Serializer\Exception\NotNormalizablevalueException can actually test, that the rethrow really happened.

Reply:
oh and you are using sprintf and in that you use concatination. But the $e->getMessage() should also be processed using another %s

Reply:
and please change the %s::%s to %s::$%s